karma.conf.js

```diff
@@ -52,7 +52,7 @@ module.exports = function(config) {
 		colors: true,
 		logLevel: config.LOG_INFO,
 
-		browsers: ['PhantomJS2'],
+		browsers: ['Chrome'],
```

Comment:
It's nice that we already support both `Chrome` and `PhantomJS2`. I find Phantom less intrusive during development, specially if you are doing TDD. 

Maybe add a comment above saying we've got both options available and the developer can then decide which one to use.